esmvaltool/cmorizers/obs/cmorize_obs_noaacobe.py

```diff
@@ -0,0 +1,67 @@
+"""ESMValTool CMORizer for NOAA Cobe data.
+
+Tier
+    Tier 3: restricted dataset.
+
+Source
+    https://climate.mri-jma.go.jp/pub/ocean/cobe-sst2/
+
+Last access
+    20200520
+
+Download and processing instructions
+    TODO
```

Comment:
@soufianekar Coud you provide this information?

Reply:
https://climate.mri-jma.go.jp/pub/ocean/cobe-sst2/

Reply:
ftp://ftp.cdc.noaa.gov/Datasets/COBE2/sst.mon.mean.nc